tests/integration/oup/test_oup_sftp.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pathlib
+from io import open
+
+import pytest
+from common.sftp_service import SFTPService
+
+
+@pytest.fixture
+def oup_client_fixture():
+    yield SFTPService(dir="upload/oup")
+
+
+def test_list_oup_files(oup_client_fixture: SFTPService):
+    data_files = os.listdir(pathlib.Path().resolve().__str__() + "/data/oup")
+    assert len(oup_client_fixture.list_files()) == len(data_files)
+
+
+def test_get_oup_file(oup_client_fixture: SFTPService):
+    data_path = pathlib.Path().resolve().__str__() + "/data/oup"
```

Comment:
why we need this? we have `pytest-datadir`

Reply:
This test does not make sense in the end. Because it is to check does the local ftp server has the same amount of files or not